spec/strategy_spec.rb

```diff
@@ -43,7 +43,7 @@ def sign_into_cas(username, password)
 
     it 'should redirect to sign-in' do
       response.should be_redirect
-      response.should redirect_to(new_user_session_url)
+      response.should redirect_to(new_user_session_url, allow_other_host: true)
```

Comment:
I think in this case, we probably don't want the `allow_other_host: true` parameter because this is [the RSpec `redirect_to` matcher](https://relishapp.com/rspec/rspec-rails/v/2-4/docs/matchers/redirect-to-matcher).